### PR TITLE
Play cards only with double left click

### LINKED ISSUE #310: Double middle-click on cards in hand puts them into play.
Like double left-clicking a card, if you double middle-click a card in hand (say, trying to look at the full card text via middle-click), it will put it into play.

### LINKED ISSUE #340: "setAutoConnect" in settings cache has typo'd function call
My bad! Introduced in https://github.com/Daenyth/Cockatrice/pull/289, spotted by @basicer.

```
settings->value("server/auto_connect", attemptAutoConnect ? 1 : 0);
```

should be

```
settings->setValue("server/auto_connect", attemptAutoConnect ? 1 : 0);
```

I don't believe the parent function `setAutoConnect` is actually called anywhere...but I'll admit to being a little clueless about the settings cache.

### PR DESCRIPTION
**Pull request by Fizztastic**
Fix #310

### CODE REVIEW

**Comment by ZeldaZach** [comment]
Tested and reviewed the code. Perfect! :shipit:

**Comment by ctrlaltca** [comment]
Is this really related to #340?

**Comment by woogerboy21** [comment]
Yeah I think its #310

**Comment by ZeldaZach** [comment]
Didn't notice the fixed ticket. It's #310

**Comment by Daenyth** [comment]
You need to amend the commit message to say 310 otherwise github will close 340

File: cockatrice/src/carditem.cpp [hunk 1]
```diff
@@ -358,13 +358,15 @@ void CardItem::mouseReleaseEvent(QGraphicsSceneMouseEvent *event)
 
 void CardItem::mouseDoubleClickEvent(QGraphicsSceneMouseEvent *event)
 {
-    if (settingsCache->getDoubleClickToPlay()) {
-        if (revealedCard)
-            zone->removeCard(this);
-        else
-            playCard(event->modifiers().testFlag(Qt::ShiftModifier));
+    if (event->buttons() == Qt::LeftButton){
```
Comment (Daenyth) [inline cockatrice/src/carditem.cpp]: You could do one `if` here with `&&`

**Comment by ZeldaZach** [comment]
I amended it for him

**Pull request merged**